### PR TITLE
support reseting of accounts of LDAP users

### PR DESCRIPTION
**Pull request by blizzz**
* [x] testing
* [x] add documentation

I didn't add a check for the state of ldap_write_support as it can be turned on or off afterwards, while typically you should have LDAP in place. It is a bit overspecific, as there could be other apps adding write functionality (but there isn't one, to my knowledge).

### CODE REVIEW

File: src/views/GroupSettings.vue [hunk 1]
```diff
@@ -28,6 +28,10 @@
 			{{ t('user_retention', 'Users are deleted when they did not log into their account within the given number of days. This will also delete all files of the affected users.') }}
 		</p>
 
+		<p v-if="ldapBackendEnabled" class="settings-hint">
+			{{ t('user_retention', 'Users from LDAP are deleted locally only, unless the LDAP write support app is enabled. When still available on LDAP, users will reappear.') }}
```
Comment (nickvergessen) [inline src/views/GroupSettings.vue]:
```suggestion
			{{ t('user_retention', 'Users from LDAP are deleted locally only, unless the LDAP write support app is enabled. When still available on LDAP, users will reappear after a new login.') }}
```

Comment (blizzz) [reply src/views/GroupSettings.vue]: Nope, login is not required. A search op that'll yield the user is enough.

**Comment by nickvergessen** [comment]
@blizzz Merge?

**Comment by schiessle** [comment]
@nickvergessen when can we expect a new release with this fix?

**Comment by nickvergessen** [comment]
>  when can we expect a new release with this fix?

Has been released now @schiessle

**Pull request merged**